Lib/fontTools/mtiLib/__init__.py

```diff
@@ -1146,12 +1146,33 @@ def expectendswith(self, s):
 		return line
 
 def build(f, font, tableTag=None):
+	"""Parse a Monotype font layout file.
```

Comment:
it doesn't just parse but it also builds OT layout tables from it (GPOS, GSUB, GDEF) as it parses along.